src/installer.js

```diff
@@ -0,0 +1,94 @@
+const os = require('os');
+const core = require('@actions/core');
+const tc = require('@actions/tool-cache');
+const path = require('path');
+const fs = require('fs');
+
+export async function getK6(versionSpec, osArch = 'amd64') {
```

Comment:
Same question as before.... in main.js this code is run:
```js
    if (!osArchitecture) {
      osArchitecture = 'amd64';
    }
```
So what is the point of setting it here too?

Reply:
I guess there is not.  I will remove it.